creations/.eslintrc.json

```diff
@@ -9,7 +9,7 @@
     "rules": {
       "react/prop-types": 0,
       "indent": ["error", 4],
-      "linebreak-style": 1,
+      // "linebreak-style": 1,
```

Comment:
@UmangLadha  Why is this commented ?
What problem was it causing ?

Reply:
it's giving the problem of semicolon I have solved reading this article.
https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems

Reply:
Although semicolon are good to have, its okay for now.